app/api/users.js

```diff
@@ -0,0 +1,76 @@
+// User creation and authentication with password hashing.
+// Code sourced from Zybooks 11.12 https://learn.zybooks.com/zybook/UMASSCOMPSCI326RichardsAcademicYear2020/chapter/11/section/12
+const jwt = require("jwt-simple");
+const User = require("../models/user");
+const router = require("express").Router();
+const bcrypt = require("bcrypt-nodejs");
+const saltRounds = "10";
+
+const secret = "secret";
+
+router.post("/user", (req, res) => {
+    bcrypt.hash(req.body.password, saltRounds, null, (err, hash) => {
+        let newUser = new User({
+            username: req.body.username,
+            password: hash,
+            status: req.body.status
+        });
+
+        newUser.save((err) => {
+            if (err) return next(err);
+            res.sendStatus(201); // New user created
+        });
+    });
+});
+
+router.post("/create", (req, res) => {
+    let testUser = new User({
+        username:   "big",
+        password:   "man",
```

Comment:
I figured it was a test account so it didn't matter

Reply:
Do you want to use the test account for anything after creating it? If you don't, it's fine as-is; if you do, you'll need to hash it or you won't be able to authenticate with the account (because the password in the database will never match the hashed password that you compare it to when a user tries to login)

Reply:
Up to you, I could hash it if we need it in the future